### PR TITLE
Keep secrets on locked chest

### PR DESCRIPTION
**Pull request by SubAt0m1c**
Map info config fix maybe?

### CODE REVIEW

File: odinmain/src/main/kotlin/me/odinmain/features/impl/dungeon/CanClip.kt [hunk 1]
```diff
@@ -103,7 +103,7 @@ object CanClip : Module(
                 else -> return
             }
 
-            if (line) Renderer.draw3DLine(pos1, pos2, color = Color.RED)
+            if (line) Renderer.draw3DLine(pos1, pos2, color = Color.RED, depth = true)
```
Comment (freebonsai) [inline odinmain/src/main/kotlin/me/odinmain/features/impl/dungeon/CanClip.kt:106]: should depth be true there? seems nicer to be able to see it through blocks

**Review by freebonsai — APPROVED** [review]
I mean the code kinda sucks but its what it needs to be so i understand, tell me if you want anything more or if i should just merge

**Comment by SubAt0m1c** [comment]
I’ll probably remake the line at some point if I get a better idea of how
to handle it. I don’t think I have anything else to do on this pr.

Seeing the line from everywhere through walls seemed rather annoying to me.
I could make it a toggle, but I don’t think seeing where to clip when youre
not near the block enough to see it is any useful.

**Comment by freebonsai** [comment]
fair

**Pull request merged**